tenseal/binding.cpp

```diff
@@ -27,6 +28,8 @@ void bind_plain_tensor(py::module &m, const std::string &name) {
         .def("shape", &type::shape)
         .def("strides", &type::strides)
         .def("size", &type::size)
+        .def("batch", &type::batch)
+        .def("__len__", &type::size)
```

Comment:
Most tensor libraries seems to return the shape[0] in len(tensor). Should we implement this the same way?

Reply:
Isn't this already implemented? or I'm missing something

https://github.com/OpenMined/TenSEAL/blob/master/tenseal/cpp/tensors/plain_tensor.h#L233

Reply:
Oh yeah, my bad.